app/assets/stylesheets/views/_taxons.scss

```diff
@@ -132,7 +127,7 @@
     }
 
     .subsection-list-item a {
-      text-decoration: none;
+      text-decoration: underline;
```

Comment:
What happens if you remove this block of CSS altogether? Presumably the default styling is `underline` ?